docs/source/docker/getting_started/first_steps.rst

```diff
@@ -257,6 +257,24 @@ Now that everything is in place, let's configure and run a simple job.
         }
     )
 
+     """Optionally, You can use this code to track the state of the compute worker and only continue if it has finished."""
+    last_log = ("", "")
+    while True:
+        state, message = client.get_compute_worker_state_and_message(scheduled_run_id=scheduled_run_id)
+
+        # Print the state and message if either is new. You can adapt this log as you like, e.g. also print the time.
+        if (state, message) != last_log:
+            print(f"Compute worker run is now in {state=} with {message=}")
+
+        # Break if the scheduled run is in one of the end states.
+        if state in [DockerRunScheduledState.CANCELED, DockerRunState.ABORTED, DockerRunState.COMPLETED, DockerRunState.FAILED]:
+            break
+
+        # Wait before polling the state again
+        time.sleep(30)  # Keep this at 30s or larger to prevent rate limiting.
```

Comment:
this loop and the time should be controlled by us and not by the user. 
it should be more a function like
```
watch_job( myScheduleId, (state, message)=>{
# state and/or message changed
})
```

Reply:
I discussed this with @IgorSusmelj. Both of use see this point, that from an API protection perspective, we should not make this too obvious. 
However, we also see it from a power user perspective, that will want full control over what happens. E.g. they have their own logging, want to make API calls based on it, add timeouts, print also if the state stayed the same, ...

Thus we would keep this.

Reply:
I am hardcore against it. Please set up a meeting between us 3.